client/src/AttendanceView/AttendanceView.js

```diff
@@ -186,12 +186,13 @@ export function AttendanceView() {
               {t('inputAttendance')}
             </Button>
           </div>
-          <p>
+          <div>
```

Comment:
This is to fix 👇 

<img width="762" alt="Screen Shot 2021-10-04 at 3 26 23 PM" src="https://user-images.githubusercontent.com/32123313/135869465-7a89734b-b49f-4a8d-b452-907f9f8db83b.png">